proto/repo.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+import "proto/context.proto";
+
+package repo;
+
+message CreateRepoRequest {
+  // The request context.
+  context.RequestContext request_context = 1;
+
+  // The name of the repo, i.e. "buildbuddy".
+  string name = 2;
+
+  // The organization that the repo should be created under ex. "buildbuddy-io".
+  // If this is empty, the repo will be created under the authenticated Github
+  // User's account.
```

Comment:
could we require this to always be set (client picks either repo or puts in user acct name)?

Reply:
There's a subtle distinction between `organization` and `owner`.

I decided to go with `organization` because it I need to feed it into this method which has takes either an organization or an empty string if you want to create it for the authenticated user instead https://github.com/google/go-github/blob/c8c34d01f65e98954a092333edf4ecdf3120c36e/github/repos.go#L428

I can switch to `owner` if you prefer, but in that case I'd also need a boolean / enum telling me whether or not the owner is an organization (Github weirdly has different endpoints for creating a [repo under an organization](https://docs.github.com/en/free-pro-team@latest/rest/repos/repos?apiVersion=2022-11-28#create-an-organization-repository) or under the [authenticated user](https://docs.github.com/en/free-pro-team@latest/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-for-the-authenticated-user) which I believe is the root of this complexity).

Reply:
alright i'm happy with your simple approach for now, we can revisit if it needs more later :)

Reply:
And yes, the org endpoint blows up if you try to pass the username as the org name - that was the first thing I tried haha